wifi/models.py

```diff
@@ -5,40 +5,50 @@
 
```

Comment:
Also, I think your editor have screwed up the indentation.  Can you please fix it.